.circleci/config.yml

```diff
@@ -191,6 +196,24 @@ jobs:
       - capture_test_results
       - capture_test_logs
 
+  acceptanceTestsGoQuorum:
+    parallelism: 1
+    executor: acceptance_tests_executor
+    steps:
+      - prepare
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: GoQuorum Acceptance Tests
+          no_output_timeout: 40m
+          command: |
+            ./gradlew --no-daemon acceptanceTestsGoQuorum
+      - store_artifacts:
+          path: build/goquorum-at
+          destination: goquorum-at-artifacts
+      - store_test_results:
+          path: build/goquorum-at/openjdk-latest/reports/xml-report
```

Comment:
why openjdk-latest in the path?

Reply:
because I manually ran for that variant and passed that in the export path for the docker container. however, I have a TODO left there about it.

@macfarla @pinges should we run the GoQ AT against only one variant or all variants?